NetCasbin/Abstractions/IEnforcer.cs

```diff
@@ -1,29 +1,15 @@
-// Copyright 2019 The casbin Authors. All Rights Reserved.
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//      http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
+using System;
 using System.Collections.Generic;
 using System.Threading.Tasks;
 
-namespace NetCasbin
+namespace NetCasbin.Abstractions
 {
-
     /// <summary>
     /// IEnforcer is the API interface of Enforcer
     /// </summary>
+    [Obsolete("The interface will be changed a lot at next mainline version, you can see https://github.com/casbin/Casbin.NET/issues/56 to know more information.")]
```

Comment:
So this pr is just an inner modification and doesn't affect the public API, why you emphasize this api will change.

Reply:
If we follow this issue #56. The `IEnforcer` at the future version will be like the `ICoreEnforccer` now. They will be completely different.